test/irmin/test_tree.ml

```diff
@@ -85,7 +85,20 @@ let test_bindings _ () =
       "Bindings are reported in lexicographic order" [ "a"; "aa"; "ab"; "b" ]
   in
   (* [Tree.list] returns all keys in lexicographic order *)
-  Tree.list tree [] >|= (List.map fst >> check_sorted)
+  let* () = Tree.list tree [] >|= (List.map fst >> check_sorted) in
+
+  (* Make sure that a [Hash (Some repo, _)] node with a cached value doesn't
+     produce a list of pruned children. See #1720 *)
```

Comment:
```suggestion
     produce a list of pruned children. *)
```
I think there is no need to link to the PR, as there are no more info in the PR than in the test, no?